Lib/test/test_inspect/test_inspect.py

```diff
@@ -48,6 +48,7 @@
 from test.test_inspect import inspect_fodder2 as mod2
 from test.test_inspect import inspect_stringized_annotations
 from test.test_inspect import inspect_deferred_annotations
+from test.test_typing import EqualToForwardRef
```

Comment:
We try really hard to unwire different test cases, if possible - maybe it would be better to import this object from somewhere else?